src/store/conversations/conversations.reducers.ts

```diff
@@ -582,6 +582,26 @@ const selectIsMessagesError = createSelector(
   },
 );
 
+const selectIsLastAssistantMessageEmpty = createSelector(
+  [selectSelectedConversations],
+  (conversations) => {
+    return conversations.some((conv) => {
+      const lastMessageIndex = conv.messages.length - 1;
+      const lastMessage = conv.messages[lastMessageIndex];
+      return (
+        lastMessage.role === 'assistant' && lastMessage.content.length === 0
+      );
+    });
+  },
+);
+
+const selectIsNotModelTypeModelInSelectedConversations = createSelector(
```

Comment:
maybe to rename to selectNotModelConversations?

Reply:
Thank you for suggestion.
Constant name have been changed to the selectNotModelConversations.